crates/wash/src/cli/cmd/dev/mod.rs

```diff
@@ -493,3 +502,29 @@ async fn stop_dev_session(
 
     Ok(())
 }
+
+/// Checks whether a wasmCloud host is up by trying to get its inventory. It will repeat after
+/// waiting for `backoff` duraction in case of error. It will fail after `timeout` duration if the
+/// inventory could still not be retrieved.
+pub async fn is_host_up(
```

Comment:
I have a slight preference for a simple loop here over the inline function, but it's ok as-is

Reply:
I'm going to go ahead and merge this, and if you feel the urge it can be a followup 😄